From 1b922ac853cd468a71549b7b95e2c0392f02a36e Mon Sep 17 00:00:00 2001
From: Subbaraya Sundeep <sbhatta@marvell.com>
Date: Wed, 27 Mar 2019 11:51:09 +0530
Subject: [PATCH 086/386] octeontx2-af: Remove unnecessary error message

When a default DMAC rule cannot be installed by
rvu_npc_install_ucast_entry then it is not an error because
user might have called "ifconfig ethx hw ether" before
interface is up which is a valid scenario. Hence removed
the error message.

Change-Id: I110c472b0ed1a2530376f619f8a0310f5dd5b238
Signed-off-by: Subbaraya Sundeep <sbhatta@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/6391
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c b/drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c
index 7b0f5dd2f7f3..b0f0f32ce530 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c
@@ -432,8 +432,7 @@ void rvu_npc_install_ucast_entry(struct rvu *rvu, u16 pcifunc,
 	req.match_id = action.match_id;
 	req.flow_key_alg = action.flow_key_alg;
 
-	if (rvu_mbox_handler_npc_install_flow(rvu, &req, &rsp))
-		dev_err(rvu->dev, "could not install default rule\n");
+	rvu_mbox_handler_npc_install_flow(rvu, &req, &rsp);
 }
 
 void rvu_npc_install_promisc_entry(struct rvu *rvu, u16 pcifunc,
-- 
2.17.1

